src/hoverboard-app.html

```diff
@@ -383,25 +322,7 @@ <h3 class="location">{$ location.short $}</h3>
             </paper-menu-button>
           </app-toolbar>
 
-          <app-toolbar class="hero-block" layout start vertical center-justified>
-            <plastic-image
-              class="hero-image"
-              srcset="[[hero.background.image]]"
```

Comment:
I don't see you've removed this from properties
   hero: {
            type: Object,
            statePath: 'hero',
            observer: '_heroChanged'
          },

haven't you?